howto/troubleshoot/troubleshoot-application-creation.md

```diff
@@ -0,0 +1,17 @@
+You might encounter the following issues when creating an application.
+
+### Application manifest
+
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+> Is there an automatic way to create a manifest for an application?
+
+No. The application manifest describes necessary metadata on top of the APK, which AMS needs. You can simplify the manifest to only contain the `name` field, but you will lose a lot of control about how your application is being executed.
```

Comment:
This is not true. Simplifying the manifest yaml file with only `name` field declared would fail in application creation. The minimal manifest file would at least include 
- name
- instance-type
or 
- name
- resources